build/configs/rtl8721csm/rtl8721csm_make_bin.sh

```diff
@@ -130,11 +132,15 @@ function concatenate_binary()
 		cp $BOOT_PATH/km4_boot_all.bin $BINDIR/km4_boot_all.bin
 		cat $BINDIR/xip_image2_prepend.bin $BINDIR/ram_2_prepend.bin $BINDIR/psram_2_prepend.bin > $BINDIR/km4_image2_all.bin
 		cat $GNUUTL/km0_image2_all.bin $BINDIR/km4_image2_all.bin > $BINDIR/km0_km4_image2.bin
+		${TOP_PATH}/os/tools/mkbinheader.py $BINDIR/km0_km4_image2.bin kernel $SECURE_HEADER_SIZE
```

Comment:
Samsung header is not board specific. How about calling it from Makefile.unix after MAKE_BOARD_SPECIFIC_BIN?

Reply:
Yes, but each binary names can be different.